lib/rucio/core/replica.py

```diff
@@ -2060,7 +2151,8 @@ def __cleanup_after_replica_deletion(scope_name_temp_table, scope_name_temp_tabl
                                              rse_id=rse_id, files=[sn._asdict() for sn in chunk], session=session)
 
     if dids_to_delete:
-        session.query(scope_name_temp_table).delete()
+        stmt = delete(scope_name_temp_table)
+        session.execute(stmt)
         session.execute(insert(scope_name_temp_table), [sn._asdict() for sn in dids_to_delete])
```

Comment:
https://github.com/rucio/rucio/pull/6591/files#r1549891959

Reply:
Changed to:

```
        values = [sn._asdict() for sn in dids_to_delete]
        stmt = insert(scope_name_temp_table)
        session.execute(stmt, values)
```